js/header.js

```diff
@@ -1,9 +1,37 @@
-document.onscroll = function() {shrinkHeader()};
 header = document.getElementsByTagName('header')[0];
+
 function shrinkHeader() {
-    if (document.body.scrollTop > 50 && document.body.clientWidth >= 1200 || document.documentElement.scrollTop > 50 && document.body.clientWidth >= 1200) {
+    if ((document.documentElement.scrollTop || document.body.scrollTop) > 50 || document.body.clientWidth <= 1275) {
```

Comment:
What's the reason for this change?

Reply:
I thought `documentElement` was too new but it seems pretty widely available. `document.body.scrollTop` kept giving back 0 scroll.

Reply:
Weird. It's definitely worked before.

Reply:
My chrome may have recently updated or something. It seems like `document.body.scrollTop` got deprecated at some point.